### DISCUSSION TITLE
When I try to fix both face and hands, it quite often turned fingers and some other parts into face.

### DISCUSSION THREAD

**Discussion by NagaoMono** [Bing-su/adetailer#238] [Q&A] [answered]
I have tried different order/combo of model and detection model confidence threshold, no matter what I have adjusted, it is just heads everywhere. What can I do to prevent this? Thanks everyone for helping 🙏🏻

Here I have 2 example, you can see their hands became small faces.
![tmpr69d25ht](https://github.com/Bing-su/adetailer/assets/140643778/23b51ccf-eabf-4d2f-8e18-1c4635b02176)
![tmpwfxh4hic](https://github.com/Bing-su/adetailer/assets/140643778/bd66325a-de1f-4b62-9f4c-aa3862458674)

**Comment by songib** [comment, marked as answer]
Try to use the Positive prompt for just the hands mine goes like this `(Beautiful Perfect Hand, Detailed:1.3)` It will fix the body or face the problems.

**Reply by NagaoMono** [reply]
Thank you, I have tested several times in this way, so far the issue didn't repeat anymore :)

**Comment by rpillala** [comment]
I also put the hand detailing 1st and face 2nd, then it refines the hands first and no longer detects those areas as faces.

**Reply by NagaoMono** [reply]
Thank you very much, let me try in this way too 🙏🏻
